Pod/Classes/CollectionViewHelpers.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CollectionViewHelpers.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 12/6/16.
+//
+//
+
+import Foundation
+
+extension UICollectionView {
+    public final func registerReusableCell<T>(_ cell: ReusableCell<T>) where T: UICollectionViewCell {
```

Comment:
Thats a stellar idea though I'm not sure it would work because of the generics. You need a generic per cell type.